src/js/profile/mobile/widget/Drawer.js

```diff
@@ -62,6 +62,8 @@
 				 */
 
 				self.options.dragEdge = 0.05;
+				self.options.width = 0.90 * window.screen.width;
```

Comment:
according to my checks in other aps menu takes about 75% of screen width

Reply:
Done.